src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
```

Comment:
set [default](https://github.com/mate-academy/layout_search-bar-airbnb#:~:text=default%20font%2Dweight%20must%20be%20300) font weight and size

compare font styles(when you input something) with the mock